vertical-pod-autoscaler/pkg/admission-controller/certs.go

```diff
@@ -92,6 +119,36 @@ func (cr *certReloader) load() error {
 	return nil
 }
 
+func (cr *certReloader) reloadWebhookCA() error {
+	client := cr.mutatingWebhookClient
+	webhook, err := client.Get(context.TODO(), webhookConfigName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	if webhook == nil {
+		return fmt.Errorf("webhook not found")
+	}
+	if len(webhook.Webhooks) > 0 {
+		currentBundle := webhook.Webhooks[0].ClientConfig.CABundle[:]
+		base64CurrentBundle := base64.StdEncoding.EncodeToString(currentBundle)
+		newBundle := readFile(cr.clientCaPath)
+		base64NewBundle := base64.StdEncoding.EncodeToString(newBundle)
+		// make sure clientCA actually changed
+		if base64CurrentBundle == base64NewBundle {
+			klog.V(2).InfoS("Client CA did not change, skipping patch")
+			return nil
+		}
+		klog.V(2).InfoS("New client CA found, reloading and patching webhook")
+		patch := []byte(fmt.Sprintf(`{"webhooks":[{"name":"%s","clientConfig":{"caBundle":"%s"}}]}`, webhookName, base64NewBundle))
+		_, err2 := client.Patch(context.TODO(), webhookConfigName, types.StrategicMergePatchType, patch, metav1.PatchOptions{})
+		if err2 != nil {
+			return err2
```

Comment:
Any reason to use `err2`?

```suggestion
		_, err := client.Patch(context.TODO(), webhookConfigName, types.StrategicMergePatchType, patch, metav1.PatchOptions{})
		if err != nil {
			return err
```